fe2-android/app/src/main/java/com/github/dedis/popstellar/model/objects/digitalcash/TransactionObject.java

```diff
@@ -217,7 +214,7 @@ public int getIndexTransaction(PublicKey publicKey) {
     String hashPubkey = publicKey.computeHash();
```

Comment:
```suggestion
    String hashPubKey = publicKey.computeHash();
```
Maybe to be consistent with new convention ?